node/node.d.ts

```diff
@@ -2244,102 +2244,108 @@ declare module "crypto" {
 declare module "stream" {
     import * as events from "events";
 
-    export class Stream extends events.EventEmitter {
+    class internal extends events.EventEmitter {
```

Comment:
@vvakame I didn't have good ideas of naming and took the idea from the implementation of module "assert". If you think _Stream would be better, I will change it :)